.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
+!geocoder_tester
+!Pipfile*
+!run_qwant.sh
+!conftest.py
```

Comment:
These files are explictly ADDed in the Dockerfile.
Why not simply put python-related files here: 
```
__pycache__
*.pyc
*.pyo
*.pyd
.Python
```

Reply:
I also have lots of tests results in the directory and I'd prefer not to have to include them.

the best way would be to have `!geocoder_tester/**.py` but it does not seems to work